CHANGELOG.md

```diff
@@ -216,6 +216,7 @@ extension interfaces. `module.Manager.Modules` is now of type `map[string]interf
 
 ### Bug Fixes
 
+* (x/group) [#](https://github.com/cosmos/cosmos-sdk/pull) Fix wrong address set in `EventUpdateGroupPolicy`.
```

Comment:
missed the pr number here 🙈, it's not too bad, as it gets removed from here as soon we tag a final v0.47 or a v0.46.8.